layouts/partials/head.html

```diff
@@ -5,7 +5,7 @@
 <!-- multilingual SEO optimizations -->
 {{ if .IsTranslated }}
 {{ range .AllTranslations }}
-<link rel="alternate" hreflang="{{.Lang}}" href="{{ .RelPermalink | absLangURL }}">
+<link rel="alternate" hreflang="{{.Lang}}" href="{{ .RelPermalink | relURL }}">

```

Comment:
Shouldn't this be `relLangURL`? Or can you explain the issue?

Reply:
I didn't noticed any difference 🤷‍♂️

It's for sure with `absLangUrl` doesn't work 😊, and if you see the original theme repo, they've used `relURL`
https://github.com/gethugothemes/bigspring-light/blob/248923a6c7e0a6853b560412be81cffc36f0c373/layouts/partials/header.html#L8

Reply:
To be honest I don't know why, but the only working is `relURL` when you're navigating the other language.
`relLangURL` works correctly when you're in the English language.

See with `relLangURL` (when you are not on the English page)
<img width="332" alt="Screenshot 2022-12-13 alle 12 53 40" src="https://user-images.githubusercontent.com/48283112/207311003-c055d515-fcce-457a-9030-54b29b9b530a.png">